obm-autoset/README.md

```diff
@@ -0,0 +1,46 @@
+#RackHD OBM Setting Auto Set Tool
+Tool to apply OBM setting for each compute node discovered by RackHD ora. Save the trouble to manually apply OBM setting node by node.
+
+##Installation and execution
+clone this repository.

```

Comment:
User only need to download this script (or this folder), there is no need to clone the whole repository.